### ISSUE TITLE
Script ignores github gist code snippets

### ISSUE THREAD

**Issue by arptoth** [gautamdhameja/medium-2-md#8]
Excellent code! Congrats! Only what I noticed that it ignores github gists. It would be wonderful if you would add it. Thanks!

**Comment by gautamdhameja** [comment]
After some initial analysis, I found out that GitHub gists are not part of exported HTML content from Medium. They seem to be rendered on the fly when browsing Medium posts on the site.
Closing this issue.

**Comment by vivainio** [comment]
No, they are part of the exported html files, but it's actually a script tag. Those could be changed to other embedding syntaxes.

Search for "Github gists" here https://towardsdatascience.com/converting-medium-posts-to-markdown-for-your-blog-5d6830408467

Just wrote about my experience using the tool: https://dev.to/vivainio/saving-my-stuff-from-medium-3hhm

**Comment by gautamdhameja** [comment]
Thanks for the pointer, @vivainio. I'll try that out.
